fashrevwall/twitter_client/TwitterClient.py

```diff
@@ -60,7 +62,27 @@ def get_images_by_hashtag(self, hashtag, n):
                 print "This tweet doesn't contain an image."
                 continue
             try:
+                #in the future, for speeding up the process if number of tweets is extremely high: http://stackoverflow.com/questions/7943233/fast-way-to-discover-the-row-count-of-a-table
+                conn = psycopg2.connect(database = "fashrevwall", host="127.0.0.1", port="5432") #opens a conection to the DB
```

Comment:
It is not a good idea to have this inside `try` because that `try` was written to capture problems when ingesting the tweet in lines 83-86. If the connection fails (for example, the DB doesn't exist), then the exception won't be captured and program will brake or become unstable without the user necessarily knowing why.

Also hardcodiing database, host and port is not a good idea, because this works in your computer but might not work in heroku when it's operational if it uses another port or whatever. These things are already configured in settings.py.